### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by ryanmrubin**
Whaddyasay?

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -2,31 +2,23 @@
...
----------------------------------
-Chris Williams <chris@nitron.org>
----------------------------------
-
 OVERVIEW
 =========
 
-django-cas-provider is a consumer for the `Central Authentication
-Service <http://jasig.org/cas>`_. It supports CAS version 1.0. It allows
+django-cas-provider is a consumer for `my.crossway.org <https://my.crossway.org>`_. It supports CAS version 1.0. It allows
```
Comment (adamzap) [inline README.rst]: Should this be `django-cas-consumer`?

File: README.rst [hunk 2]
```diff
@@ -2,31 +2,23 @@
...
 
 INSTALLATION
 =============
 
-To install, run the following command from this directory:
-
-    	``python setup.py install``
-
-Or, put cas_consumer somewhere on your Python path.
+Install this package.
```
Comment (adamzap) [inline README.rst]: This section isn't super helpful 😉

File: README.rst [hunk 3]
```diff
@@ -93,4 +85,4 @@ Some CAS server implementations require that GET variables arrive in a specific
 		(CAS_TICKET_LABEL, None),
 		(CAS_SERVICE_LABEL, None)))
 
-The ``odict`` package can be installed via pypi and can also be found via the [Plone Archetypes SVN repo](https://svn.plone.org/svn/archetypes/AGX/odict/). Any Class that implements both the ``update()`` and ``items()`` ``dict`` methods should work though.
\ No newline at end of file
+The ``odict`` package can be installed via pypi and can also be found via the [Plone Archetypes SVN repo](https://svn.plone.org/svn/archetypes/AGX/odict/). Any class that implements both the ``update()`` and ``items()`` ``dict`` methods should work though.
```
Comment (adamzap) [inline README.rst]: Probably safe to remove the SVN stuff here

File: README.rst (hunk 2 above)
Comment (ryanmrubin) [inline README.rst]: hehe.

**Pull request merged**